### ISSUE TITLE
Breaking Issue: EDITORCONFIG_LINTER_RULES rules file not found

### ISSUE THREAD

**Issue by omusavi** [github/super-linter#858]
**Describe the bug**
```
2020-10-15 17:24:47 [INFO  ]   Successfully found:[GITHUB_TOKEN]
2020-10-15 17:24:47 [INFO  ]   --------------------------------------------
2020-10-15 17:24:47 [INFO  ]   Gathering user validation information...
2020-10-15 17:24:50 [FATAL ]     -> EDITORCONFIG_LINTER_RULES rules file (/action/lib/.automation/.ecrc) doesn't exists. Terminating...
```

No changes were made, this is likely caused by #854 

**To Reproduce**
Run super linter, do not specify EDITORCONFIG_LINTER_RULES 

**Expected behavior**
No errors

**Comment by omusavi** [comment]
Note I had VALIDATE_EDITORCONFIG=false as well.

For now I set our CI rules to pin to v3.13.0 and that has fixed the issue

**Comment by PhunStyle** [comment]
Same issue here, The PR you referenced #854 fixes the ESLint issue i had in #852 but introduces this new error.

I feel like this has to do with the fact that the script tries to validate all default linter rules found here: https://github.com/github/super-linter/tree/master/TEMPLATES -- note how .ecrc is not one of them.

**Comment by GaboFDC** [comment]
I think this is related to #824 that added using the file, but didn't add a default.

@admiralAwkbar we need some tests to catch these cases, this broke a lot of flows and was completely unseen. I'm working on a fix atm.
